src/components/UserProfile/UserProfilePopover.ts

```
@@ -0,0 +1,7 @@
+import { paperClasses, Popover, styled } from '@mui/material';
+export const UserProfilePopover = styled(Popover)(({}) => ({
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)